### ISSUE TITLE
Followers do not sync up 

### ISSUE THREAD

**Issue by EmilJacobs** [snarfed/bridgy-fed#1582]
I observe that many followers on Bluesky to my Mastodon account, do not sync up to my follower count, while the accounts on Bluesky DO follow the bridge.

I suspect that this is because these accounts first followed me and then the bridge, or did so at more or less the same time when things are still being setup. I just tested this hypothesis by having a Bluesky account I setup earlier and which me on Mastodon, unfollow me and immediately refollow, after which it *did* sync.

I'm not sure if the same issue occurs from the fediverse to Bluesky.

Expected behaviour:
To have the bridge synchronise the follower count both ways, even after initial setup (following the bridge).

**Comment by snarfed** [comment]
Thanks for the nudge! I found one bug that probably contributed to this; fixed now.

However, keeping follower counts exactly correct isn't really a goal of the bridge. Bridging all interactions, profiles, etc, is definitely a priority, but follower counts, not so much.

Sorry for the bad news!

**Comment by snarfed** [comment]
^ added this to the docs.

**Comment by EmilJacobs** [comment]
Why actually? I mean, I respect your decision, but I don't quite understand the motivation. Is it requiring too much work?

**Comment by snarfed** [comment]
The short answer is, yes. Comparing it to both my [current roadmap](https://github.com/search?q=owner%3Asnarfed+label%3Anow+is%3Aopen&type=issues) and the [other 180 open issues here](https://github.com/snarfed/bridgy-fed/issues), this just isn't a high priority. Sorry.

**Comment by snarfed** [comment]
Having said that, PRs are always welcome if you want to tackle it!

**Comment by Tamschi** [comment]
Related issue: (#1102

**Comment by snarfed** [comment]
Agreed @Tamschi, this is probably the same issue. Merging in there.

**Comment by snarfed** [comment]
Duplicate of #1102
